src/index.html

```diff
@@ -11,7 +11,183 @@
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You? 
+
+          <input 
+            type="number" 
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth: 
+
+          <input 
+            type="date" 
+            name="date-of-birth"
+          >
+        </label>
+
+        <label class="form-field">
+          I accept the term of the agreement 
+
+          <input 
+            type="checkbox" 
+            name="term"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset-form">
+        <legend>Registration</legend>
+
+        <label class="form-field">
+          E-mail:  
+          <input 
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label class="form-field">
+          Password:  
+          <input 
+            type="password"
+            name="password"
+            minlength="6"
+            maxlength="30"
+          >
+        </label>
+
+
+
+      </fieldset>
+
+      <fieldset class="fieldset-form">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          Do you love cats? 
+
+          <label>
+            <input 
+              type="radio"
+              name="love-cats"
+              value="yes"
+            >
+            Yes
+          </label>
+
+          <label>
+            <input 
+              type="radio"
+              name="love-cats"
+              value="no"
+            >
+            No
+          </label>
+        </div>
+
+        <label class="form-field">
+          What is your favorite color? 
+
+          <input 
+            type="color"
+            name="color"
+          >
+        </label>
+
+        <label class="form-field">
+          What time do you go to bed? 
+
+          <input 
+            type="time" 
+            name="time-go-bed"
+          >
+        </label>
+
+        <label class="form-field">
+          What are your favorite brand of cars? 
+          
+          <select 
+            name="cars" 
+            required 
+            multiple
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="form-field">
+          How do you rate our work? 
+
+          <input 
+            type="range" 
+            name="rate-work"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset-form">
+        <legend>Additional info:</legend>
+
+        <label class="form-field">
+          Comments:  
+          <textarea 
+            name="comments" 
+            cols="30" 
+            rows="2">
+          </textarea>
```

Comment:
```suggestion
            rows="2"
          ></textarea>
```
виправ [textarea](https://mate.academy/learn/css-advanced/forms#/video/885)